lib/presentation/engage/bible_series/widgets/scripture_body.dart

```diff
@@ -81,7 +81,8 @@ Widget book(Scripture script, List<int> sortedVerses) {
 Widget copyright(ScriptureBody script) {
   String attribution = script.properties.attribution;
   String version = script.properties.bibleVersion;
-  String text = "$version@\n$attribution";
+  String text =
+      "Scripture quotations taken from $version\n$attribution\nUsed by permission. All rights reserved worldwide";
```

Comment:
We should be able to make this change in Firebase right?

Reply:
The version and the attribution can be changed. This is just the format that NIV recommends on their website.

Reply:
I'm wondering if we can change bible_version in Firebase from `The Holy Bible, New International Version® NIV®` to `Scripture quotations taken from The Holy Bible, New International Version® NIV®` and something similar for attribution.

Reply:
Just so that we have full control of this in Firebase

Reply:
I see what you mean. That makes sense, I'll go make those changes.

Reply:
Do we still need this MR then?

Reply:
just for the "Used by permission. All rights reserved worldwide"